### ISSUE TITLE
Life wind data

### ISSUE THREAD

**Issue by kebekus** [Akaflieg-Freiburg/enroute#215]
It might be possible to access life wind data from the Norwegian weather service…

**Comment by spuckebaer** [comment]
SimpleVFR use an API key from OpenWeatherMap....might be an easy work arround?

**Comment by kebekus** [comment]
True. But since I program the app alone, all this will take a long time…

**Comment by kebekus** [comment]
Data available at this URL:

https://developer.yr.no/
